wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -1,6 +1,6 @@
 {% templatetag openblock %} load i18n wagtailcore_tags {% templatetag closeblock %}
 
-<header class="header">
+<header class="w-header">
```

Comment:
No need to change this one, it is the project template and hence does not need to use the Wagtail admin styles.